tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -291,12 +314,35 @@ func (rb *PSATestSuite) TestPSA() {
 		})
 
 		rb.Run("Testcase - Validate if members with roles "+role+"can add/edit/delete labesl from admin created namespace", func() {
-			rb.ValidatePSA(role)
+			if role == roleCustomCreateNS {
+				customRole = true
+			} else {
```

Comment:
Tiny nit: Optionally, consider removing else for cleaner code. By defaulting customRole to false and changing only if needed, the code could be slightly more readable.

```go
customRole = false
if role == roleCustomCreateNS {
    customRole = true
}
```

Reply:
agreed

Reply:
thanks Andrew! nice suggestion. yep the keyword is initialized at L257.